packages/nodejs/src/__tests__/config.test.ts

```diff
@@ -130,7 +130,7 @@ describe("Configuration", () => {
 
     describe("with logPath option with file specified", () => {
       beforeEach(() => {
-        config = new Configuration({ logPath: "/other_path/appsignal.log" })
+        config = new Configuration({ logPath: "/other_path/foo.log" })
       })
 
       it("uses the overwritten path", () => {
```

Comment:
nitpick: maybe this test name could now say "uses the overwritten path **but changes the file name**"?